From 44d274bf1014b1418ad9c03c7938c42b4861f3f3 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:49:11 -0400
Subject: [tools] perf: Include test-stackprotector-all.c in test-all

Message-id: <1408953210-26343-64-git-send-email-jolsa@redhat.com>
Patchwork-id: 88786
O-Subject: [PATCH RHEL7.1 BZ1133083 063/322] perf tools: Include test-stackprotector-all.c in test-all
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit bc32358b569247ceabba14dab297070a8fb255c4
Author: Jiri Olsa <jolsa@redhat.com>
Date: Thu Nov 28 12:33:01 2013 +0100

description
===========
We're missing test-stackprotector-all.c in test-all check, adding it.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/feature-checks/test-all.c b/tools/perf/config/feature-checks/test-all.c
index 799865b..3397329 100644
--- a/tools/perf/config/feature-checks/test-all.c
+++ b/tools/perf/config/feature-checks/test-all.c
@@ -89,6 +89,10 @@
 # include "test-timerfd.c"
 #undef main
 
+#define main main_test_stackprotector_all
+# include "test-stackprotector-all.c"
+#undef main
+
 int main(int argc, char *argv[])
 {
  main_test_libpython();
@@ -110,6 +114,7 @@ int main(int argc, char *argv[])
  main_test_backtrace();
  main_test_libnuma();
  main_test_timerfd();
+ main_test_stackprotector_all();
 
  return 0;
 }
-- 
1.7.1